### ISSUE TITLE
htmd lib for xtc writing is not updated

### ISSUE THREAD

**Issue by giadefa** [Acellera/htmd#623]
from htmd.ui import *
mol = Molecule('3ptb')
mol.coords = np.zeros((mol.numAtoms, 3, 100), dtype=np.float32)
mol.write('test.xtc')
mol2 = Molecule('3ptb')
mol2.read('test.xtc')
mol2.coords.shape

this simple code shows it

**Comment by j3mdamas** [comment]
It is detected on Travis, but not being caught for some reason: https://api.travis-ci.org/v3/job/356992777/log.txt

I think I know why this happened, it's an incompability with htmd lib. Will fix it tomorrow and check also why it was not caught bby Travis.

**Comment by j3mdamas** [comment]
OK, there were two issues:
- the buffering of the tests in Travis was broken (subprocesses and the parent process buffer unsynced on Travis...), so I was thinking there was no problem, because I wasn't seeing the outputs.
- htmdlib was converted into two branches (stable and master), so we could keep support for both when there's a divergence between rel-X.X.x and master HTMD branches. I did not merge the master htmdlib into stable when I released HTMD.

- [ ] fix buffering for tests
- [ ] merge htmdlib master into stable
